os/sys/rtimer.h

```diff
@@ -84,6 +85,7 @@ typedef uint64_t rtimer_clock_t;
 
 #define RTIMER_CLOCK_MAX           ((rtimer_clock_t)-1)
 #define RTIMER_CLOCK_LT(a, b)      (RTIMER_CLOCK_DIFF((a),(b)) < 0)
+#define RTIMER_CLOCK_LT_OR_EQ(a,b) (RTIMER_CLOCK_DIFF((a),(b)) <= 0)
```

Comment:
code style (and consistency with line above): missing space after comma in macro arguments

Reply:
That macro is no longer needed